locales/en/plugin__dfr-console.json

```
@@ -32,10 +32,12 @@
   "Deployment Name": "Deployment Name",
   "An error occurred": "An error occurred",
   "Attach": "Attach",
-  "Create NamespaceStore ": "Create NamespaceStore ",
-  "Represents an underlying storage to be used as read or write target for the data in the namespace buckets.": "Represents an underlying storage to be used as read or write target for the data in the namespace buckets.",
-  "Namespace store name": "Namespace store name",
-  "A unique name for the namespace store within the project": "A unique name for the namespace store within the project",
+  "Create Data Resource": "Create Data Resource",
```

let's make it "resource" for now, ideally we make it capital when it is a custom resource, but here "Data resource" is not any CR, will need to discuss with UX once later...
```suggestion
  "Create Data Resource": "Create Data resource",
```